res/values/strings.xml

```diff
@@ -755,33 +755,33 @@
     <string name="qrscan_title">Scan QR Code</string>
     <string name="qrscan_hint">Hold your camera over the QR code.</string>
     <string name="qrscan_failed">QR code could not be decoded</string>
-    <string name="qrscan_ask_join_group">Do you want to join the group \"%1$s\"?</string>
+    <string name="qrscan_ask_join_group">Do you want to join the group %1$s?</string>
```

Comment:
why remove the quotes here?

Reply:
Words are not a placeholder. It does not require the use of quotation marks.

Reply:
> Words are not a placeholder.

i do not really understand that.

using quotes or sth. like that for object names inserted to a sentence is a quite usual pattern. the quotes make clear what the name actually is, where it starts and where it ends. some advantages are recognizability and also tricking users with weird names is harder when there are quotes.

so, if we go for changing that, there must be very good reasons, i cannot see them :)

Reply:
Could this cause a problem as variables only represent a name? I do not think so. When someone calls you by your name, they don't use quotation marks. You can think of it that way too.